### PR TITLE
Allow different styles for wac

### PR DESCRIPTION
**Pull request by shhsu**

### CODE REVIEW

File: src/gulpfile.js [hunk 1]
```diff
@@ -73,8 +73,18 @@ gulp.task('generate-resjson', (cb) => {
     runSequence(['generate-resjson-json', 'generate-resjson-interface'], 'merge-localized-json', cb);
 });
 
+gulp.task('generate-angular-cli-json', (_) => {
```
Comment (jhkimnew) [inline src/gulpfile.js]: Please move this new task definition above the the task definition of generate-powershell to match the order of runSequence(['generate-angular-cli-json', 'generate-powershell', 'generate-svg', 'generate-resjson'], cb).

File: src/themes/themes.wac.css [hunk 2]
```diff
@@ -0,0 +1,256 @@
+﻿.red {
```
Comment (jhkimnew) [inline src/themes/themes.wac.css:1]: Please put some comment why and how you created this style and the difference from the original style.

File: src/gulpfile.js (hunk 1 above)
Comment (shhsu) [reply src/gulpfile.js]: done

File: src/themes/themes.wac.css (hunk 2 above)
Comment (shhsu) [reply src/themes/themes.wac.css:1]: Not sure about this. It's a copy of theme.blue.css but that's not a relevant information from this point onwards and the difference will only grow.

File: src/angular-cli.site.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (jhkimnew) [inline src/angular-cli.site.json:1]: 'site' is ambiguous. Can we rename with angular-cli.default.json or something like that?

Comment (shhsu) [reply src/angular-cli.site.json:1]: 'default' is too generic. There are 2 scenarios of this repo: wac extension and standalone website. Hence I call the scenario site

File: src/gulpfile.js [hunk 4]
```diff
@@ -73,8 +73,18 @@ gulp.task('generate-resjson', (cb) => {
     runSequence(['generate-resjson-json', 'generate-resjson-interface'], 'merge-localized-json', cb);
 });
 
+gulp.task('generate-angular-cli-json', (_) => {
+    var override = process.argv.slice(3).find(function(s, _, __) { return s.startsWith('--env=') })
+    var scenario = override ? override.split('=').pop().split('.')[0] : 'site'
+    return gulp.src(['angular-cli.template.json', `angular-cli.${scenario}.json`])
```
Comment (jhkimnew) [inline src/gulpfile.js]: put some comment here for readability.

**Review by jhkimnew — APPROVED** [review]
Approving with some suggestions. Please check them and update if you agree.

**Pull request merged**